tests/system/Config/BaseConfigTest.php

```diff
@@ -271,18 +271,27 @@ public function testBadRegistrar(): void
         $this->assertSame('bar', $config->foo);
     }
 
+    /**
+     * @psalm-suppress UndefinedClass
+     */
     public function testDiscoveryNotEnabledWillNotPopulateRegistrarsArray(): void
     {
         /** @var MockObject&Modules $modules */
         $modules = $this->createMock(Modules::class);
         $modules->method('shouldDiscover')->with('registrars')->willReturn(false);
-
         RegistrarConfig::setModules($modules);
+
         $config = new RegistrarConfig();
 
         $this->assertSame([], $config::$registrars);
+
+        // Reset Modules Config.
+        RegistrarConfig::setModules(new Modules());
```

Comment:
this is already done in setUp

Reply:
Yes, but if the test method moves to the last, the mock object remains.
Since we are using static properties, we have to reset them again and again.

Reply:
Then it should be added in tearDown also so if the test fails, resetting will still be done.

Reply:
Added tearDown() and so on.